pkg/setting/setting_feature_toggles.go

```diff
@@ -10,65 +13,188 @@ import (
 	"gopkg.in/ini.v1"
 )
 
+type FeatureToggleState string
+
+const (
+	AlphaState   FeatureToggleState = "alpha"   // anything can change without warning
+	BetaState    FeatureToggleState = "beta"    // shape looks right, but still evoloving
+	StableState  FeatureToggleState = "stable"  // stable and may soon become a build-in feature
+	MergedState  FeatureToggleState = "merged"  // feature no longer requires an explicit flag
+	RemovedState FeatureToggleState = "removed" // the feature was not merged and no longer has any effect
+)
+
 var (
 	featureToggleInfo = promauto.NewGaugeVec(prometheus.GaugeOpts{
 		Name:      "feature_toggles_info",
 		Help:      "info metric that exposes what feature toggles are enabled or not",
 		Namespace: "grafana",
 	}, []string{"name"})
+)
+
+type FeatureToggleInfo struct {
+	Id          string             `json:"id"`   // the key listed in grafana.ini
+	Name        string             `json:"name"` // user
+	Description string             `json:"description"`
+	AliasIds    []string           `json:"aliasIds,omitempty"` // ids that will also work (support spelling changes and product renames)
```

Comment:
Is this really needed? I'm worried that it makes things come complicated. Accidentally enabling/disabling feature is extremely annoying :)

Reply:
AliasIds?  -- not necessary now now.  It was copied from the "registry" structure we use everywhere in the frontend.  This essentially allows us to fix/clean/rename things without breaking existing configs.

But agree we can leave it off